### ISSUE TITLE
txt export plugin broken

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#240]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/240) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

The txt plugin was seemingly broken by the Py2/Py3 support work.
A file now looks like this:

```
#!txt

Track name: b'friday'
Index  Location   Lat (+-90)  Lon (+-180)  Flightlevel  Pressure (hPa)  Leg dist. (km)  Cum. dist. (km)  Comments
    0  b'Kathman'     27.697       85.359        0.000        1013.250             0.0              0.0  b''
    1  b'A'           27.370       84.980      490.000         121.684            52.1             52.1  b''

```

The b'' ist newly introduced by the Py3 changes. This additional 3 characters break the formating of the file, preventing it from being read in, causing an Exception that crashed the MSS client.

The code needs to be modified to remove the b''.

The code should be modified not to crash the client even in the presence of an unforeseen exception. Loading a flighttrack can be simply aborted and should not crash the application.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Switched text file plugins to utf-8 encoding

The Py3 compatability broke the ASCII output. To fix this, it is more
sensible to simply switch to UTF-8, which is better to begin with.

Fix issue #240

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Made file io plugins use more resilient against errors

See issue #240
